vm/error_test.go

```diff
@@ -448,3 +450,21 @@ func checkErrorTraces(t *testing.T, index int, evaluated Object, expectedTraces
 		t.Fatalf("At test case %d: Expect traces to be:\n%s \n got: \n%s", index, joinedExpectedTraces, joinedTraces)
 	}
 }
+
+func checkFuzzifiedErrorMsg(t *testing.T, index int, evaluated Object, expectedErrMsg string) {
+	t.Helper()
+	err, ok := evaluated.(*Error)
+	if !ok {
+		t.Fatalf("At test case %d: Expect Error. got=%T (%+v)", index, evaluated, evaluated)
+	}
+
+	if fuzzifyMessage(err.message) != expectedErrMsg {
+		t.Fatalf("At test case %d: Expect error message to be:\n  %s. got: \n%s", index, expectedErrMsg, err.message)
+	}
+}
+
+func fuzzifyMessage(message string) string {
+	re, _ := regexp2.Compile("(?<=#<[a-zA-Z0-9_]+:)[0-9]{12}(?=[ ]>)", 0)
+	fuzMsg, _ := re.Replace(message, "XXXXXXXXXXXX", 0, -1)
```

Comment:
Same suggestion as for `verifyStringObject`

Reply:
I'll do as that.